test/corrupt.js

```diff
@@ -2,9 +2,11 @@ import fixtures from 'webtorrent-fixtures'
 import parseTorrent from '../index.js'
 import test from 'tape'
 
-test('exception thrown when torrent file is missing `name` field', t => {
-  t.throws(() => {
-    parseTorrent(fixtures.corrupt.torrent)
-  })
+test('exception thrown when torrent file is missing `name` field', async t => {
+  try {
+    await parseTorrent(fixtures.corrupt.torrent)
+  } catch (e) {
+    t.ok(e instanceof Error)
```

Comment:
shouldn't there be a t.error after the await to fail the test?

Reply:
I see, yeah I'll add that